src/components/Hero/Hero.tsx

```diff
@@ -48,84 +40,137 @@ export default function Hero(props: React.PropsWithChildren<HeroProps>) {
     heading,
     heroType,
     image,
+    modifiers = [],
     locationDetails,
     subHeaderText,
   } = props;
 
-  let heroModifiers;
-
   if (heroType === HeroTypes.Primary) {
-    heroModifiers = ["primary"];
+    modifiers.push("primary");
   } else if (heroType === HeroTypes.Secondary) {
-    heroModifiers = ["secondary"];
+    modifiers.push("secondary");
   } else if (heroType === HeroTypes.Tertiary) {
-    heroModifiers = ["tertiary"];
+    modifiers.push("tertiary");
   } else if (heroType === HeroTypes.FiftyFifty) {
-    heroModifiers = ["50-50"];
+    modifiers.push("50-50");
+  } else if (heroType === HeroTypes.Campaign) {
+    modifiers.push("campaign");
   }
 
   if (heroType === HeroTypes.Primary && !backgroundImageSrc) {
-    throw new Error(`backgroundImageSrc required on PRIMARY heroTypes`);
+    console.warn(
+      `Warning: it is recommended to use the "backgroundImageSrc" prop for PRIMARY hero`
+    );
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.Primary && image) {
+    console.warn(
+      `Warning: the "image" prop has been passed, but PRIMARY hero will not use it`
+    );
+    modifiers.push("warning");
   }
 
-  if (backgroundImageSrc && image) {
-    throw new Error(
-      `Please only either backgroundImageSrc or image into Hero, got both`
+  if (heroType === HeroTypes.Secondary && backgroundImageSrc) {
+    console.warn(
+      `Warning: the "backgroundImageSrc" prop has been passed, but SECONDARY hero will not use it`
     );
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.Tertiary && (backgroundImageSrc || image)) {
+    console.warn(`Warning: TERTIARY hero will not use any of the image props`);
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.FiftyFifty && backgroundImageSrc) {
+    console.warn(
+      `Warning: the "backgroundImageSrc" prop has been passed, but FIFTYFIFTY hero will not use it`
+    );
+    modifiers.push("warning");
+  }
+
+  if (heroType === HeroTypes.Campaign && (!backgroundImageSrc || !image)) {
+    console.warn(
+      `Warning: it is recommended to use both "backgroundImageSrc" and "image" props for CAMPAIGN hero`
+    );
+    modifiers.push("warning");
   }
 
   if (heroType !== HeroTypes.Primary && locationDetails) {
-    throw new Error(`Please provide locationDetails only to PRIMARY heroTypes`);
+    console.warn(
+      `Warning: Please provide "locationDetails" only to PRIMARY hero`
+    );
+    modifiers.push("warning");
   }
 
-  const backgroundImageStyle = backgroundImageSrc
-    ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
-    : { backgroundColor: backgroundColor };
+  let backgroundImageStyle = {};
+  if (heroType === HeroTypes.Primary) {
+    backgroundImageStyle = backgroundImageSrc
+      ? { backgroundImage: "url(" + backgroundImageSrc + ")" }
```

Comment:
Use string interpolation for this and on line 117.

Reply:
That's the original code, but it's now cleaned up.